test/JDBC/expected/securityadmin_role-vu-verify.out

```diff
@@ -0,0 +1,1609 @@
+-- it should be able to connect to the database
+use securityadmin_db1
+go
+
+-- it should be dbo
+select current_user
+go
+~~START~~
+varchar
+dbo
+~~END~~
+
+
+-- both attribute should be true
+select rolname, rolcreaterole, rolcreatedb from pg_roles where rolname = 'securityadmin_login1'
+go
+~~START~~
+varchar#!#bit#!#bit
+securityadmin_login1#!#1#!#1
+~~END~~
+
+
+-- terminate-tsql-conn user=securityadmin_login1 password=123
+
+-- tsql
+-- only member of securityadmin
+alter server role sysadmin drop member securityadmin_login1
+go
+
+create user securityadmin_user1 for login securityadmin_login1
+go
+
+-- terminate-tsql-conn
+
+-- tsql user=securityadmin_login1 password=123
+
+-- only rolcreaterole attribute should be true
+select rolname, rolcreaterole, rolcreatedb from pg_roles where rolname = 'securityadmin_login1'
+go
+~~START~~
+varchar#!#bit#!#bit
+securityadmin_login1#!#1#!#0
+~~END~~
+
+
+-- should be able to create/drop login
+create login test_securityadmin_l1 with password ='123'
+go
+
+drop login test_securityadmin_l1
+go
+
+-- terminate-tsql-conn user=securityadmin_login1 password=123
+
+-- tsql
+-- only member of sysadmin
+drop user securityadmin_user1
+go
+
+alter server role sysadmin add member securityadmin_login1
+go
+
+alter server role securityadmin drop member securityadmin_login1
+go
+
+-- terminate-tsql-conn
+
+-- tsql user=securityadmin_login1 password=123
+
+-- both attribute should be true
+select rolname, rolcreaterole, rolcreatedb from pg_roles where rolname = 'securityadmin_login1'
+go
+~~START~~
+varchar#!#bit#!#bit
+securityadmin_login1#!#1#!#1
+~~END~~
+
+
+-- should be able to create/drop login
+create login test_securityadmin_l1 with password ='123'
+go
+
+drop login test_securityadmin_l1
+go
+
+-- should return 1
+select is_srvrolemember ('sysadmin')
+go
+~~START~~
+int
+1
+~~END~~
+
+
+-- should return 1
+select is_srvrolemember ('securityadmin')
+go
+~~START~~
+int
+1
+~~END~~
+
+
+-- terminate-tsql-conn user=securityadmin_login1 password=123
+
+-- tsql
+-- make it member of securityadmin only
+alter server role sysadmin drop member securityadmin_login1
+go
+
+alter server role securityadmin add member securityadmin_login1
+go
+
+drop user no_securityadmin_user1
+go
+
+alter server role sysadmin add member no_securityadmin_login1
+go
+
+create user securityadmin_user1 for login securityadmin_login1
+go
+
+-- terminate-tsql-conn
+
+-- tsql user=securityadmin_login1 password=123
+-- Case 2 - negatives
+-- alter server role sysadmin should give permission denied
+Alter server role sysadmin add member no_securityadmin_login1
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Current login securityadmin_login1 does not have permission to alter server role)~~
+
+
+-- alter server role securityadmin add member db roles should error out
+Alter server role securityadmin add member guest
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: role "guest" does not exist)~~
```

Comment:
This error is misleading

Reply:
since alter server role is server level and on server level guest is not present, so it is expected.